### PR TITLE
Support x86/x64 targets in the build apk command

### LINKED ISSUE #5592: No way to `flutter build` for genymotion
We just had a report in gitter about hitting ABI incompatible message with genymotion when dragging-dropping an apk built by flutter:

"When I try to install the example gallery.apk to genymotion, I still get this error msg flutter/flutter#1154 which is a closed bug. why? thanks.
I drag the gallery.apk file to genymotion. After the file transferring finished, the screen shows a error window: "An error occurred while deploying the file. INSTALL_FAILED_NO_MATCHING_ABIS"."

I looked at `flutter build apk` and there appears to be no way to produce an x86 apk from there.  Or even a fat apk which includes all the abis we know how to build for.

I'm not sure how important this is.  Maybe we should be pointing folks at Android Studio for this case?  Certainly `flutter run` knows how to handle this

### PR DESCRIPTION
**Pull request by jason-simmons**
Fixes https://github.com/flutter/flutter/issues/5592

### CODE REVIEW

**Comment by jason-simmons** [comment]
@devoncarew

**Comment by sethladd** [comment]
Is it possible to test these targets? Or, test that these new commands don't regress?

**Comment by jason-simmons** [comment]
It looks like flutter_tools doesn't yet have a test for buildApk.

@yjbanov would know more about the end-to-end tests and which device platforms have coverage

File: packages/flutter_tools/lib/src/commands/build_apk.dart [hunk 1]
```diff
@@ -215,7 +243,7 @@ class BuildApkCommand extends BuildSubCommand {
     } else {
       // TODO(devoncarew): This command should take an arg for the output type (arm / x64).
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/commands/build_apk.dart]: I think we can delete this TODO now.

**Comment by devoncarew** [comment]
lgtm

File: packages/flutter_tools/lib/src/commands/build_apk.dart (hunk 1 above)
Comment (jason-simmons) [inline packages/flutter_tools/lib/src/commands/build_apk.dart]: done

**Comment by yjbanov** [comment]
It should be easy to have a "build does not crash" smoke-test on Travis. We don't need real devices to run the build.

**Pull request merged**